src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/TypedContentTypeBuilder.cs

```diff
@@ -26,10 +26,12 @@ public void Build(FieldType contentQuery, ContentTypeDefinition contentTypeDefin
                 var partName = part.Name;
 
                 // Check if another builder has already added a field for this part.
-                if (contentItemType.HasField(partName))
+                if (contentItemType.HasMetadata(partName))
```

Comment:
This doesn't achieve the same as the HasField check did previously. DynamicContentTypeBuilder doesn't add metadata so this condition will now let multiple builders add the same field which will cause errors.

Reply:
I believe this change is causing: ArgumentOutOfRangeException: A field with the name: BagPart is already registered for GraphType: TestBag&#xA;Parameter name: Name</h2>